shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java

```diff
@@ -293,7 +293,8 @@ private void switchLaunchThemeForNormalTheme() {
     } catch (PackageManager.NameNotFoundException exception) {
       Log.e(
           TAG,
-          "Could not read meta-data for FlutterFragmentActivity. Using the launch theme as normal theme.");
+          "Could not read meta-data for FlutterFragmentActivity. Using the launch theme as normal"
+          + " theme.");
     }
   }
```

Comment:
There must be a similar change here as well for calling super. Check around line 490.

Reply:
Done. Sorry somehow I missed this.